pkg/apis/serving/v1alpha1/revision_defaults.go

```diff
@@ -16,6 +16,17 @@ limitations under the License.
 
 package v1alpha1
 
+import (
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+		// DefaultTimeout will be set if timeoutSeconds not specified.
+		DefaultTimeout = 60 * time.Second
```

Comment:
Do we document this default anywhere under `./doc/spec/`?

Reply:
there is a default for ClusterIngress.
Added a max timeout for ClusterIngress and validation to check that ClusterIngress and RevisionSpec Timeout does not exceed the max timeout.